backend-tests/tests/test_tenantadm_v2.py

```diff
@@ -68,77 +75,72 @@ def test_ok_non_sca_cards(self, clean_mongo):
         Some of these are omitted - they are in fact being rejected with:
         'Please use a Visa, MasterCard, or American Express card'
         """
+        tenant = "test.mender.io-{}".format(randstr())
```

Comment:
Why not use `str(uuid.uuid4())` like virtually everywhere else?

Alternative question: why not modify `common.randstr` to use `uuid`s, and use it everywhere?

Reply:
Yup, can do that, I pushed a new commit.